app/pages/Auction/RepairBid.js

```diff
@@ -69,7 +72,7 @@ export default class RepairBid extends Component {
       const attempt = await walletClient.getNonce({
         name: bid.name,
         address: bid.from,
-        bid: value * consensus.COIN
+        bid: value
```

Comment:
Ok one more thing: we shouldn't change the value sent to `verifyBid()` because a few lines down `rpc importNonce` actually requires a float in HNS, not an integer in dollarydoos. The HTTP endpoint `getNonce` DOES accept an integer of dollarydoos so your intuition is correct, but this will break repairing any bid that is NOT `0`.

For this reason, I think we need to keep this line as `bid: value * consensus.COIN` and up on line 66, weneed to keep `return this.verifyBid(value);`